### PR TITLE
Added sort feature lists

### PR DESCRIPTION
**Pull request by SeqLaz**
Added options to sort feature lists

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3224 #4192 Feature list sorting capability](https://app.clickup.com/t/2192114/QF-3224)

File: documentation/how-to/map-interaction.en.md [hunk 1]
```diff
@@ -24,6 +24,16 @@ On double-tap or long-press on a layer, you get offered the following possibilit
 - *Reload data* to get the current data of a layer with remote sources.
 - *Show feature list* to show all the layer's features in the identification list.
 
+If you'd like to organize the feature list on the map canvas or in the 'Show feature list' in QField, you have the following options:
+
```
Comment (suricactus) [inline documentation/how-to/map-interaction.en.md:28]:
```suggestion
## Sort layer features

If you'd like to configure the order of features in 'Show feature list' in QField, you have the following options:

```

This needs a header block.

What do you mean by "organize the feature list on the map canvas"?

**Review by suricactus — COMMENTED** [review]
Small comment posted. However, not 100% sure if the map interaction is the best place to post this? I am thinking about a new "Feature list" page where we describe this as well as long press on features?

@m-kuhn @lucienicolier @nirvn opinions?

File: documentation/how-to/map-interaction.en.md (hunk 1 above)
Comment (nirvn) [reply documentation/how-to/map-interaction.en.md:28]: Agreed.

Comment (SeqLaz) [reply documentation/how-to/map-interaction.en.md:28]: My bad, you are right is only in the Show feature list of the layes.

File: documentation/how-to/map-interaction.en.md [hunk 2]
```diff
@@ -24,6 +24,18 @@ On double-tap or long-press on a layer, you get offered the following possibilit
 - *Reload data* to get the current data of a layer with remote sources.
 - *Show feature list* to show all the layer's features in the identification list.
 
+## Sort layer features
+
+If you'd like to configure the order of features in 'Show feature list' in QField, you have the following options to preconfigure this on QGIS:
```
Comment (suricactus) [inline documentation/how-to/map-interaction.en.md:29]:
```suggestion
## Sort layer features
:material-monitor: Desktop preparation

If you'd like to configure the order of features in 'Show feature list' in QField, you have the following options to preconfigure this on QGIS:
```

We can add something like this, I think it is nice hint that you cannot change it on QField.

https://github.com/opengisch/QField-docs/blob/master/documentation/how-to/pictures.en.md?plain=1#L20

**Comment by lucienicolier** [comment]
> Small comment posted. However, not 100% sure if the map interaction is the best place to post this? I am thinking about a new "Feature list" page where we describe this as well as long press on features?
> 
> @m-kuhn @lucienicolier @nirvn opinions?

@suricactus I agree, that is more an interaction with data then with the map. Should we make a page call data-interaction?

**Comment by suricactus** [comment]
Yes, if we have map iteraction, we can have data iteraction too. Not sure where the feature list is described right now, probably it is a good spot too.

**Review by suricactus — APPROVED** [review]

**Review by suricactus — APPROVED** [review]

**Pull request merged**